jsonnet/components/prometheus-adapter-audit.libsonnet

```
@@ -0,0 +1,30 @@
+local profile(level) = {
+  apiVersion: 'audit.k8s.io/v1',
+  kind: 'Policy',
+  metadata: {
+    name: level,
+  },
+  // omit stage RequestReceived to avoid duplication of logs for both stages
+  // RequestReceived and ResponseComplete
+  omitStages: ['RequestReceived'],
+  rules: [{ level: level }],
+};
+
+
+{
+  volume: {
+    name: 'audit-log',
+    path: '/var/log/adapter',
+  },
+  cm: {
```

nit: can we expand it? cm => configMap?
```suggestion
  configMap: {
```

---

I purposely kept it to `cm` to avoid the confusion with a `configMap` that is used elsewhere in the project.